### PR TITLE
[WIP][Remove] unnecessary PackageReferences from IME and AddIn .csproj files

### PR DESCRIPTION
**Pull request by samatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the CDP4-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/CDP4-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
A simple test to check whether the bulk of PackageReferences can be removed from the IME and AddIn project since these are already dependencies of the Composition proejct. I did not check whether the installer still works, but that should straightforward.

### CODE REVIEW

**Comment by samatrhea** [comment]
@lxatrhea this is ready to be reviewd. It seems to be working on my end but also needs independent verification by @MartinFijneman once we have provided him with the installer(s). I am having trouble getting in built in Release mode

**Review by alexatrhea — CHANGES_REQUESTED** [review]
Needs to be rebased on #504

**Review by alexatrhea — APPROVED** [review]

File: BasicRdl/Views/Dialogs/BooleanParameterTypeDialog.xaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:dx="http://schemas.devexpress.com/winfx/2008/xaml/core"
              xmlns:dxlc="http://schemas.devexpress.com/winfx/2008/xaml/layoutcontrol"
-             xmlns:items="clr-namespace:CDP4CommonView.Items;assembly=CDP4CommonView"
+             xmlns:items="clr-namespace:CDP4CommonView.Items;assembly=CDP4Composition"
```
Comment (samatrhea) [inline BasicRdl/Views/Dialogs/BooleanParameterTypeDialog.xaml:7]: why do these changes appear as part of this PR? they were taken care of in a previous commit?

**Review by samatrhea — COMMENTED** [review]
a bit challenging to review due to the amount of files that have changed with should probabaly not be part of this PR since they were already merged in a previous approved PR

File: BasicRdl/Views/Dialogs/BooleanParameterTypeDialog.xaml (hunk 1 above)
Comment (lxatrhea) [reply BasicRdl/Views/Dialogs/BooleanParameterTypeDialog.xaml:7]: It was rebased and these commits just show up here.

**Pull request closed without merging**